File: 1-two-sum/two-sum.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* twoSum(int* nums, int numSize, int target, int* returnSize)
{
    int *result=(int*)malloc(sizeof(int)*2);
    for(int i=0;i<numSize-1;i++)
    {
        for(int j=i+1;j<numSize;j++)
        {
            if(nums[i]+nums[j]==target)
            {
                *result=i;
                *(result+1)=j;
                *returnSize=2;
            
            }
        }
    }return result;
}

File: 1-two-sum/two-sum.java
import java.util.ArrayList;
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int i;
        int[] res = new int[2];
        for( i=0;i<nums.length-1;i++)
{
        for(int j=i+1;j<nums.length;j++)
        {
            if(nums[i]+nums[j]==target)
            {
                res[0]=i;
                res[1]=j;
                break;
            }
            
        }
    }
        return res;
    }
    
}